api-outbound.yaml

```diff
@@ -128,7 +128,10 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/AssetEvent'
+              oneOf:
+                # an asset needs either a hostname OR an IP
+                - $ref: '#/components/schemas/AssetEventWithIP'
+                - $ref: '#/components/schemas/AssetEventWithHostname'
```

Comment:
Question: why either or? `anyOf` allows both to validate, and if an asset has both, I don't think submitting both should be an error condition. Otherwise, we may need to change upstream service code.

Reply:
This is my confusion over how oneOf vs anyOf works. Will fix.